dotCMS/src/main/java/com/dotmarketing/business/APILocator.java

```diff
@@ -416,6 +416,15 @@ public static ContentletAPI getContentletAPI() {
 		return (ContentletAPI)getInstance(APIIndex.CONTENTLET_API_INTERCEPTER);
 	}
 
+	/**
+	 * This is the contentletAPI which will be used to perform ES related operations
+	 * normal {@link ContentletAPI} logic.
+	 *
+	 * @return The {@link ContentletAPI} class.
+	 */
+	public static ESContentletAPIImpl getESContentletAPIImpl()
+	{return (ESContentletAPIImpl)getInstance(APIIndex.CONTENTLET_API);}
+

```

Comment:
This method is not necessary. You just need to call the already existing `getContentletAPI()`

Reply:
this has been addressed